drivers/storage/portworx/util/util.go

```diff
@@ -701,7 +721,8 @@ func GetPortworxConn(sdkConn *grpc.ClientConn, k8sClient client.Client, namespac
 		return nil, fmt.Errorf("failed to get endpoint for portworx volume driver")
 	}
 
-	endpoint := pxService.Spec.ClusterIP
+	// note, using symbolic name for the `endpoint`, as SSL certificates won't have K8s service IP
+	endpoint := PortworxServiceName + "." + namespace + ".svc.cluster.local"
```

Comment:
I guess probably this is why the unit test is timing out, this messed up the mock calls there. I feel it's kind of risky to make this change here without test coverage, and if we can make it backward compatible.

Reply:
Thanks for the heads-up -- will take a look